### ISSUE TITLE
Provide an example about editing / persisting data

### ISSUE THREAD

**Issue by rogerpadilla** [ngrx/example-app#25]
The current example application, only does query data (read only); in an usual application, editing data through a form is a common escenario. 

How should be called a Web service for retrieving data to populate a grid? How to integrate that with the store and dispatcher? 

How should be called a Web service to fill a form in 'edit'  mode? How to integrate that with the store and dispatcher? 

In short, a real app commonly persists its data using some backend server.

**Comment by MikeRyan52** [comment]
Agreed. We'll try to think of ways to improve the example to include web service API calls and interacting with live services.

Stay tuned!

**Comment by Codermar** [comment]
I am kind of interested in contributing to this. Is this open to contribution?
I was thinking along the lines of doing a new section with several case scenarios for persisting data:
One option is to do something around the google books and other is to use something like the Angular heroes example converted to ngrx. 

The interesting case scenario I had to deal with recently was an entity (Issues in this case) with a read-only view which showed associated items in different tabs i.e. Workarounds/Engineering Needs/Notes etc. We have a view route with child routes and at the same time the typical route to create and edit the issue information. 
I imagine this is actually a pretty common scenario. (or the simpler Read-only View + CRUD without the associations)
The challenge of course would be to make it small enough to be didactic and relatively simple without omitting the important cases.
 
Any thoughts on this?

**Comment by tobiasmuecksch** [comment]
Is there any article or tutorial describing how to do this? I have hoped to learn this particular question by examining this example.

**Comment by agugan** [comment]
@MikeRyanDev Any updates??
